packages/suite/src/support/messages.ts

```diff
@@ -1953,6 +1953,18 @@ export default defineMessages({
         defaultMessage: 'Custom delimiter',
         id: 'TR_IMPORT_CSV_MODAL_DELIMITER_CUSTOM',
     },
+    TR_IMPORT_CSV_FROM_FILE: {
+        defaultMessage: 'Import from file',
+        id: 'TR_IMPORT_CSV_FROM_FILE',
+    },
+    TR_IMPORT_CSV_FROM_TEXT: {
+        defaultMessage: 'Import as text',
+        id: 'TR_IMPORT_CSV_FROM_TEXT',
+    },
+    TR_IMPORT_CSV_FROM_TEXT_BUTTON: {
+        defaultMessage: 'Import',
+        id: 'TR_IMPORT_CSV_FROM_TEXT_BUTTON',
+    },
```

Comment:
There is the `IMPORT_CSV` key with the same value, let's use it to save our translators some work.

Reply:
done, [338cca7](https://github.com/trezor/trezor-suite/pull/10331/commits/338cca7524d208c082812c6d24c5b67e90e66944)